scripts/src/fodt/fix_ignored.py

```diff
@@ -0,0 +1,166 @@
+            logging.info(f"Found {len(critical_keywords)} critical keywords.")
+            return critical_keywords
+        else:
+            logging.error(f"Failed to download {url}.")
+        return None
+
+    def replace_ignored(self, txt: str, critical: set[str]) -> str:
+        if critical:
+            replacement = ("This keyword is not supported by OPM Flow but would change "
+                    "the results if supported so the simulation will be stopped.")
+        else:
+            replacement = ("This keyword is not supported by OPM Flow but has no effect "
+                    "on the results so it will be ignored.")
+
+        variants = [
+            "This keyword is ignored by OPM Flow and has no effect on the simulation but is documented here for completeness.",
+            "This keyword is ignored by OPM Flow and has no effect on the simulation;",
+            ", it is ignored by OPM Flow and has no effect on the simulation but is documented here for completeness.",
+            "This keyword is ignored by OPM Flow and has no effect on the simulation but",
+            "; hence, <text:s/>this keyword is ignored by OPM Flow and has no effect on the simulation but is documented here for completeness."
+        ]
+        found_variant = False
+        for variant in variants:
+            if variant in txt:
+                if variant[0:2] == ", ":
+                    replacement = ". " + replacement
+                elif variant[-1] == ";":
+                    replacement = replacement[:-1] + ";"
+                elif variant[-3:] == "but":
+                    replacement = replacement[:-1] + " but"
+                elif variant[0] == ";":
+                    replacement = "; hence t" + replacement[1:]
+                txt = txt.replace(variant, replacement)
+                found_variant = True
+                break
+        if not found_variant:
+            txt = replacement
+        return txt
+
+# USAGE:
+#
+#   fodt-fix-ignored-keywords
+#
+# DESCRIPTION:
+#
+#  Scan all keyword documents for certain phrases regarding ignored keywords. If found, change the
+#  phrase to a new phrase. To determine the new phrase, the keyword is checked for criticality.
+#
+#  - Download file with information about critcal/non-critical keywords here:
+#
+#    https://raw.githubusercontent.com/OPM/opm-simulators/master/opm/simulators/utils/UnsupportedFlowKeywords.cpp
+#
+#  - Phrases to look for:
+#    - "is ignored by OPM Flow and has no effect on the simulation"
+#
+#  - Replacement phrase for critical keywords:
+#    "This keyword is not supported by OPM Flow but would change the results "
+#       if supported so the simulation will be stopped."
+#
+#  - Replacement phrase for non-critical keywords:
+#     "This keyword is not supported by OPM Flow but has no effect on the results so will be ignored."
+#
+@click.command()
+@ClickOptions.maindir(required=False)
+def fix_ignored(maindir: str
+) -> None:
```

Comment:
Is the line break here intentional?

Reply:
No they are not :) Thanks, for the comment, I will fix this. (I have problems pushing to this branch for some reason, so need to fix this problem first)

Reply:
> I have problems pushing to this branch for some reason, so need to fix this problem first

I have updated the PR in a new PR, see #149.